reconcile/gitlab_housekeeping.py

```diff
@@ -353,6 +353,7 @@ def get_merge_requests(
             "mr": mr,
             "labels": labels,
             "label_priority": label_priotiry,
+            "priority": f"{label_priotiry} - {MERGE_LABELS_PRIORITY[label_priotiry]}",
```

Comment:
just noticed that typo: `label_priotiry`. Not new though...

Reply:
https://github.com/app-sre/qontract-reconcile/pull/3662/commits/07091194d3a2db9a46d34bf9ab861ab2d39e4feb